functions/src/definitions/common/responseUtils.ts

```diff
@@ -1155,7 +1161,14 @@ async function respondToInstance(
           await userSnap.ref.update({
             numSubmissionsRemaining: FieldValue.increment(1),
           })
-          await sendTextMessage("user", from, responseText, data.id)
+          if (communityNote.downvoted) {
```

Comment:
These changes shouldn't be made in respondToInstance, it should be a separate function imo. Because respondToInstance is also called whenever subsequent users send in new instances, even when they didn't get the community note in the first place. In that case, we don't want to have this correction statement. Either that, or we need to check isCommunityNoteSent for this instance

Reply:
Also because the logic needs to set communityNote.pendingCorrection to false.

Reply:
If we using another function, can think about whether it makes sense to split out the line 1065 onwards of respondToInstance into another function because it'll be repeated in respondToInstance and correctCommunityNote.